cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStore.java

```diff
@@ -1052,6 +1059,36 @@ private Scan scanAppClasses(NamespaceId namespace) {
     return new Scan(startRow, Bytes.stopKeyForPrefix(startRow));
   }
 
+  /**
+   * Filters the plugins contained in the {@link ArtifactMeta} by using {@link #isAllowed(PluginClass)}
+   *
+   * @param artifactMeta the artifact meta
+   * @return the {@link ArtifactMeta} containing only the plugins which are not excluded
+   */
+  private ArtifactMeta filterExcludedPlugins(ArtifactMeta artifactMeta) {
```

Comment:
Call it `filterPlugins` as the method relies on the predicate `isAllow`, which can be more than exclude

Reply:
Done.